src/bayes.h

```diff
@@ -1276,7 +1276,7 @@ typedef struct modelinfo
     int         *nStates;                   /* # states of each compressed char             */
     int         *cType;                     /* whether char is ord, unord or irrev          */
     int         *weight;                    /* prior weight of each compressed char         */
-    int         isTiNeeded[20];             /* marks whether a trans prob matrix is needed  */
+    int         isTiNeeded[100];            /* marks whether a trans prob matrix is needed  */
```

Comment:
Should this not be MAX_CHAR_STATES? And not 100?

Reply:
Perhaps it would be good to make MAX_CHAR_STATES specific to standard characters and call it MAX_STD_STATES instead?

Reply:
changed to isTiNeeded[MAX_STD_STATES*3]